contributors/udayempire.md

```diff
@@ -0,0 +1,53 @@
+# UDAY KUMAR
+
+📍 **Bhopal, Madhya Pradesh, India, 462022**
+📧 [kuday0726@gmail.com](mailto:kuday0726@gmail.com)  
+🌐 [My Website](https://udayempire.tech) | [GitHub](https://github.com/udayempire) | [LinkedIn](https://www.linkedin.com/in/uday-kumar-empire07/)
+
+---
+
+## 📌 SUMMARY
+I'm a **self-taught programmer** specializing in **Full Stack Development**. Passionate about **hackathons, open-source contributions**, and building impactful projects.
+
+---
+
+## 💼 WORK EXPERIENCE
+### **Devsthanam, Co-Founder** (Nov 2023 - Present)
```

Comment:
Liked the current format, can add a description to your experience by adding what you did there, what is the tangible outcome your work did (x% increase in efficiency or pipeline or y% decrease in latency).